driver/razerkbd_driver.c

```diff
@@ -11,6 +11,7 @@
 #include <linux/hid.h>
 #include <linux/dmi.h>
 
+#include "linux/input-event-codes.h"
```

Comment:
Shouldn't this be `#include <linux/input-event-codes.h>` ?

Reply:
It definitely should. `clangd` must have added that when I wasn't looking. I will fix that